### PR TITLE
Fix RE_URL_AS_PARAM (don't treat string like 'http+something') as a URL

### PR DESCRIPTION
**Pull request by tomislater**
Fix issue #2

### CODE REVIEW

**Comment by tomislater** [comment]
I would also add some tests

File: searchcmd/search_engines.py [hunk 1]
```diff
@@ -35,7 +37,7 @@ class SearchEngine(object):
     BASE_URL = None
     RE_INTERNAL_DOMAINS = None
 
-    RE_URL_AS_PARAM = re.compile(r'http[^&]+')
+    RE_URL_AS_PARAM = re.compile(r'ht{2}[ps]+:/{2}[^&\s]*')
```
Comment (jimmyppi) [inline searchcmd/search_engines.py]: That rex would match for example httsssp://

I think `r'https?://[^&\s]+'` would be more precise.

**Comment by jimmyppi** [comment]
Yes, tests would be good. Do you want to do that too?

**Comment by tomislater** [comment]
Yeah, I can do it

File: searchcmd/search_engines.py (hunk 1 above)
Comment (tomislater) [inline searchcmd/search_engines.py]: Right

**Comment by tomislater** [comment]
Hmm, I found another issue in tests; `get_command` method does not work properly on OS X. I also can fix it.

**Comment by tomislater** [comment]
On OS X you get other commands from `list_available_commands` function.

You don't have `hwclock` for example.

**Comment by tomislater** [comment]
Ok, now I am going to write tests for `RE_URL_AS_PARAM`

**Comment by tomislater** [comment]
The code is ready to review.

File: tests/test_cmdextract.py [hunk 2]
```diff
@@ -136,6 +141,14 @@
...
...c.openbsd', u'fuser', u'less', u'red', u'ntfs-3g.probe', u'ntfs-3g', u'ntfsdump_logfile', u'ntfstruncate', u'ntfsck', u'nc', u'bunzip2', u'ntfscluster', u'ntfsfix', u'dbus-daemon', u'bzdiff', u'dbus-cleanup-sockets', u'ulockmgr_server', u'bzegrep', u'chacl', u'lsblk', u'bzexe', u'dmesg', u'ntfs-3g.secaudit', u'ntfsinfo', u'ed', u'ntfsls', u'lesspipe', u'bzcat', u'netcat', u'ntfs-3g.usermap', u'ls', u'sh.distrib', u'dd', u'mountpoint', u'gzip', u'false', u'udevadm', u'pidof', u'pwd', u'mktemp', u'chgrp', u'grep', u'ss', u'sed', u'znew', u'rmdir', u'plymouth', u'ln', u'df', u'ip', u'cat', u'hostname', u'kill', u'mt-gnu', u'lsmod', u'stty', u'rbash', u'zcmp', u'login', u'fgrep', u'zcat', u'nisdomainname', u'zless', u'true', u'egrep', u'cp', u'readlink', u'uname', u'running-in-container', u'dnsdomainname', u'gunzip', u'touch', u'dir', u'which', u'mt', u'sh', u'tar', u'cpio', u'kmod', u'zdiff', u'vdir', u'bash', u'mkdir', u'sleep', u'domainname', u'uncompress', u'chown', u'chmod', u'rm', u'date', u'plymouth-upstart-bridge', u'gzexe', u'zegrep', u'tempfile', u'zfgrep', u'ps', u'sync', u'zgrep', u'ypdomainname', u'zforce', u'dash', u'echo', u'mv', u'zmore', u'mknod', u'run-parts']  # noqa
```
Comment (jimmyppi) [inline tests/test_cmdextract.py:144]: Like this, good with a static list of available commands when testing.

**Pull request merged**